crowd_sim/envs/crowd_sim.py

```diff
@@ -91,6 +130,8 @@ def generate_random_human_position(self, human_num, rule):
         :param rule:
         :return:
         """
+        ## Geneate static obstacles first
+        self.generate_random_obstacles(self.static_obstacle_num)
```

Comment:
Instead of calling `generate_random_obstacles()` here, can we call it before the line we call `generate_random_human_position()` e.g. in `reset()` method?

Reply:
yes! we can! sorry about that! This would make the name of the method more consistent!